content/ru/docs/concepts/overview/working-with-objects/finalizers.md

```diff
@@ -0,0 +1,53 @@
+
+## Как работают финализаторы
+
+Когда вы создаете ресурс с помощью файла манифеста, вы можете указать финализаторы в поле Metadata.finalizers. Когда вы пытаетесь удалить ресурс, сервер API, обрабатывающий запрос на удаление, замечает значения в поле финализаторов и выполняет следующее:
+
+* Изменяет объект, добавляя поле метаданных.deletionTimestamp со временем начала удаления.
+* Предотвращает удаление объекта до тех пор, пока все элементы не будут удалены из его поля метаданных.finalizers.
+* Возвращает код состояния 202 (HTTP «Принято»)
+
+Контроллер, управляющий этим финализатором, замечает обновление объекта, устанавливающее метаданные.deletionTimestamp, что указывает на то, что было запрошено удаление объекта. Затем контроллер пытается удовлетворить требования финализаторов, указанных для этого ресурса. Каждый раз, когда условие финализатора удовлетворяется, контроллер удаляет этот ключ из поля финализаторов ресурса. Когда поле финализаторов очищается, объект с установленным полем deleteTimestamp автоматически удаляется. Вы также можете использовать финализаторы, чтобы предотвратить удаление неуправляемых ресурсов.
+
+Типичным примером финализатора является kubernetes.io/pv-protection, который предотвращает случайное удаление объектов PersistentVolume. Когда объект PersistentVolume используется подом, Kubernetes добавляет финализатор pv-protection. Если вы попытаетесь удалить PersistentVolume, он перейдет в состояние завершения, но контроллер не сможет его удалить, поскольку существует финализатор. Когда под перестает использовать PersistentVolume, Kubernetes очищает финализатор pv-защиты, а контроллер удаляет том.
+
+{{<note>}}
+* Когда вы удаляете объект, Kubernetes добавляет метку времени удаления для этого объекта, а затем сразу же начинает ограничивать изменения в поле .metadata.finalizers для объекта, который сейчас ожидает удаления. Вы можете удалить существующие финализаторы (удалив запись из списка финализаторов), но не можете добавить новый финализатор. Вы также не можете изменить значение deletionTimestamp для объекта после его установки.
+
+* После запроса на удаление вы не сможете воскресить этот объект. Единственный способ — удалить его и создать новый аналогичный объект.
+{{</note>}}
+
+## Ссылки на владельцев, метки и финализаторы
+
+Как и метки, [ссылки на владельца](https://kubernetes.io/docs/concepts/overview/working-with-objects/owners-dependents/) описывают отношения между объектами в Kubernetes, но используются для другой цели. Когда контроллер управляет такими объектами, как модули, он использует метки для отслеживания изменений в группах связанных объектов. Например, когда задание создает один или несколько модулей Pod, контроллер заданий применяет метки к этим модулям и отслеживает изменения во всех модулях в кластере с той же меткой.
```

Comment:
- We should leave the "Job" term in English here since users are not likely to read "задание" as intended.
- "один или несколько модулей Pod" — I don't think "модулей" is an appropriate word here. It's just "создание одного или нескольких подов".
- "изменения во всех модулях в кластере" — the same problem is here and in the next paragraph.